server/routes/validPassword.js

```diff
@@ -0,0 +1,18 @@
+const { messages } = require("../common");
+const PASSWORD_FORMAT = /^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)[a-zA-Z\d\w\W]{5,}$/;

```

Comment:
Can you DRY this up?

We already have this regex declared in `registration.js`. Maybe you can export it in that file and import it here.